[18.0][MIG] product_sale_manufactured_for: Migration to 18.0

LucasTran380381 opened the pull request:
depends: `https://github.com/OCA/sale-workflow/pull/3415`

cubells committed:
product_pricelist_supplierinfo: priceslists based in supplier info

pedrobaeza committed:
product_pricelist_supplierinfo: Several enhancements + concept fixing

* Don't depend on sales
* Extended README
* Tests focused on module specific features
* Code optimization
* Don't mix pricelist info with supplier info on criteria

pedrobaeza committed:
product_pricelist_supplierinfo: Take into account sales order date

Date order is passed by context, so we have to take this into account for
computing properly available supplierinfo records.

pedrobaeza committed:
product_pricelist_supplierinfo: Give more priority in order to min_qty

If not, you can twist the order in the form and rules won't be applied according criteria

Nikul-Chaudhary committed:
[MIG] product_pricelist_supplierinfo v11 to v12

carlosdauden committed:
[11.0][FIX] product_pricelist_supplierinfo: Add template compatibility (#436)

Previously, only worked fine with product.product.

pedrobaeza committed:
[FIX] product_pricelist_supplierinfo: Avoid false computations

If you set the value "Based on" for using supplier info, but then you change the
computation type to another one (like fixed price or discount), this code is still
acting, so we should check both fields.

pedrobaeza committed:
[FIX] product_pricelist_supplierinfo: Proper URL

eLBati committed:
Added translation using Weblate (Italian)

Commit:
[ADD] product_category_archive: new module

pedrobaeza committed:
[FIX] product_category_archive: Constraint search refinement

The problem why child_of was not working directly in the search is because there are
involved inactive records that are excluded by default by the ORM.

We need to disable that exlusion for a proper working, and it's needed because the
other domain is not taking into account possible inactive categories in the middle
of the hierarchy.

Commit:
[IMP] product_category_archive: add a test

Commit:
[IMP] product_medical: Change in layout, add multiple docs

1. The layout for the old tab MDR (now called Conformity)
   has been changed to split the fields into two categories:
   conformity-related and documents-related.

2. The two fields storing the "Declaration of Conformity" and
   the "CE Certificate Medical class", that before stored an
   attachment, now allow to store several attachments.
       To make this change, a migration script has been
   provided, because the field names have changed to adhere
   to the naming conventions.

3. The string for the field ce_certificate_medical_class_ids
   has changed from 'CE Certificate Medical class' to 'CE Certificate'.
   The translation files have been updated.

didierdonze committed:
Translated using Weblate (German)

Currently translated at 68.3% (28 of 41 strings)

Translation: product-attribute-13.0/product-attribute-13.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_medical/de/

augustodinizl committed:
Added translation using Weblate (Portuguese (Brazil))

mourad-ehm committed:
[IMP] add margin on supplier info to configure the sale price

When having price per qty at the supplier level and if you want to
base your pricelist on the supplier price, now you can
define a margin on the product.supplierinfo

ruuter committed:
[FIX] product_pricelist_supplierinfo: use _select_seller

Use standard way to select the seller.

As a result a bug in previous solution is fixed also: before this,
all supplierinfo records having product_tmpl_id set, were considered
as possible price for all product variants, even if product_id was
set also. That resulted in same price for all variants even if
supplierinfo records had different price for all variants.

claudiagn committed:
Translated using Weblate (Spanish)

Currently translated at 93.3% (14 of 15 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_pricelist_supplierinfo/es/

CarmenMiranda committed:
[FIX] product_pricelist_supplierinfo: Converting Price

If we have suppliers in two or more currencies, and we have a pricelist
with an item that is selected the "Prices based on supplier info" option
but the pricelist is in a currency different from the supplierinfo the
compute of the price get the price on the same amount but with a different
currency.
With these changes, we are converting the price to the currency on the
pricelist that is being used.
Also, a test was added to prove the previous behaviour.

eLBati committed:
IMP product_pricelist_supplierinfo allowing to filter suppliers to get prices from

ypapouin committed:
Added translation using Weblate (French)

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (20 of 20 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_pricelist_supplierinfo/fr/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

Translated using Weblate (Portuguese)

Currently translated at 50.0% (2 of 4 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_category_active/pt/

ruuter committed:
[FIX] product_pricelist_supplierinfo: ensure product exists

It is possible that this method is called on an inactive template,
in which case product_variant_id returns empty recordset and
_select_seller will raise expected singleton error.

CarmenMiranda committed:
[FIX] product_pricelist_supplierinfo: Convert price to UOM on the Sale

If we have a product with a supplier with a UOM "A" but the product is
sold with a UOM "B" the price we are getting is the same no matter the
UOM. For example:
Product with the price 1200 per Dozen and we sell a 1 unit we are
getting the price of 1200 for that unit when we should be obtaining 100.

With these changes, we are converting the price to the UOM that is being
used on the sale or the default on the product.
Also, a test was added to prove the previous behaviour.

bosd committed:
Added translation using Weblate (Dutch)

claudiagn committed:
Added translation using Weblate (Catalan)

Translated using Weblate (Catalan)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_category_active/ca/

claudiagn committed:
Added translation using Weblate (Catalan)

CarlosRoca13 committed:
[IMP] product_category_active: black, isort, prettier

CarlosRoca13 committed:
[MIG] product_category_active: Migration to v13.0

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Added translation using Weblate (Catalan)

eLBati committed:
FIX product_pricelist_supplierinfo: _get_supplierinfo_pricelist_price when date is datetime.
date may come from context

Commit:
[IMP] product_medical: views for conformity categories

The attributes for the Conformity Configuration (Medical Class,
Medical Category, PPE Category, and In Vitro Diagnostics) had no
views associated to them, so the only way of accessing them was
through the entry they have in the form for product.template.

With this PR, tree views and their actions have been added, so
that they can be reused in other modules to accessed them
from custom menu-items.

Commit:
[FIX] product_medical: minor changes in labels

- Medical Class -> Medical Device Class
- Medical Category > Drug Category

CarlosRoca13 committed:
[IMP] product_pricelist_supplierinfo: black, isort

CarlosRoca13 committed:
[MIG] product_pricelist_supplierinfo: Migration to v13.0

[UPD] Update product_pricelist_supplierinfo.pot

[UPD] README.rst

Commit:
[ADD] product_sale_manufactured_for

The functionality that was before embedded into the module
shopinvader_customer_special_product has been extracted into
its own new module.

Jira Card: BSCOS-2089

CarlosRoca13 committed:
[FIX] product_pricelist_direct_print: Error when date is not defined

When we check the price of any product, the date that we pass on the context has to be a Datetime value or None. With this change we avoid to pass a False.

Here you have the reference to the code that check that https://github.com/OCA/OCB/blob/13.0/addons/product/models/product.py#L599-L600

TT29935

product_pricelist_supplierinfo 13.0.1.0.2

kevinkhao committed:
[ADD] product_attribute_archive

dzungtran89 committed:
[IMP] product_medical: black, isort, prettier

dzungtran89 committed:
[MIG] product_medical: Migration to 14.0

chienandalu committed:
[FIX] product_pricelist_supplierinfo: public users

In the e-commerce, if we have a pricelist with a rule based on
supplierinfo for a determined supplier partner, such partner could be
not accesible for public users and the affected products could not be
reached.

TT31476

victoralmau committed:
[IMP] product_pricelist_supplierinfo: black, isort, prettier

victoralmau committed:
[MIG] product_pricelist_supplierinfo: Migration to 14.0
TT31816

[UPD] Update product_pricelist_supplierinfo.pot

[UPD] README.rst

[IMP] update dotfiles [ci skip]

rousseldenis committed:
[14.0][MIG] product_category_active

GuillemCForgeFlow committed:
[13.0][ADD]product_supplierinfo_archive

chienandalu committed:
[FIX] product_pricelist_supplierinfo: override min quantity

If we don't have a supplierinfo that below 1 unit we won't get any
seller. We want to ensure that the condition is fully ignored to get
whatever price given the other criterias.

TT33659

oca-travis committed:
[UPD] Update product_supplierinfo_archive.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

GuillemCForgeFlow committed:
[13.0][FIX]product_supplierinfo_archive: Added field on view and changed filters

tgrellety committed:
[MIG] product_category_active: Migration to 15.0

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (52 of 52 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_medical/ca/

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

TDu committed:
product_main_supplierinfo: fix default sellers for variant

If no specific record are found by the first filter. The Odoo default
method will return all active supplier info.
Which will include also supplier info record related to other specific
variant.

With this change if some supplier info related to the product
variant exist, they will be returned instead.

Also add a sort on the price to have the same behaviour than on a
purchase order line for the price selected.

pedrobaeza committed:
[FIX] product_pricelist_supplierinfo: Allow to show shop for public users

0581b669ee435de4b1c068e6883e92a1e511ea41 is not enough on v13 for
avoiding the access error, so we sudoed the whole price fetch operation.

TT36898

OCA-git-bot committed:
product_main_supplierinfo 14.0.1.0.1

JasminSForgeFlow committed:
[MIG] product_supplierinfo_archive: Migration to 14.0

JasminSForgeFlow committed:
[MIG] product_supplierinfo_archive: Migration to 15.0

OCA-git-bot committed:
[UPD] README.rst

victoralmau committed:
[MIG] product_pricelist_supplierinfo: Migration to 15.0

TT36975

victoralmau committed:
[FIX] product_pricelist_supplierinfo: Show text in the view

TT36975

product_pricelist_supplierinfo 14.0.1.1.0

Commit:
[UPD] Update product_pricelist_supplierinfo.pot

OCA-git-bot committed:
[UPD] README.rst

rousseldenis committed:
[IMP] product_packaging_type: Rename to product_packaging_level

rousseldenis committed:
[IMP] product_packaging_level: Add migration scripts

Commit:
[MIG] product_category_active: Migration to 16.0

[UPD] Update product_category_active.pot

[UPD] README.rst

product_category_active 16.0.1.0.1

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_category_active/

carlosdauden committed:
[IMP] product_pricelist_supplierinfo: New context force_filter_supplier_id

OCA-git-bot committed:
product_pricelist_supplierinfo 15.0.1.0.1

Rferri44-S73 committed:
[16.0][MIG] product_attribute_archive: Migration to 16.0

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 59.0% (13 of 22 strings)

Translation: product-attribute-15.0/product-attribute-15.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-15-0/product-attribute-15-0-product_pricelist_supplierinfo/it/

PicchiSeba committed:
Added translation using Weblate (Italian)

PicchiSeba committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_category_active/it/

[UPD] README.rst

dsolanki-initos committed:
[IMP] product_supplierinfo_archive: pre-commit stuff

dsolanki-initos committed:
[MIG] product_supplierinfo_archive: Migration to 16.0

GuillemCForgeFlow committed:
[FIX]product_pricelist_supplierinfo: fix tests

aliciagaarzo committed:
[MIG] product_main_supplierinfo: Migration to 16.0

OCA-git-bot committed:
product_pricelist_supplierinfo 15.0.1.0.2

andreagidaltig committed:
[MIG] product_pricelist_supplierinfo: Migration to 16.0

Assertion in tests involving the pricelist field in products were
removed, as that field doesn't exist anymore [1], and such assertions
were intended only to ensure computed price were propagated correctly to
that field.

[1] https://github.com/odoo/odoo/commit/9e99a9df

Commit:
[UPD] Update product_pricelist_supplierinfo.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_pricelist_supplierinfo/

andreagidaltig committed:
[FIX] product_pricelist_supplierinfo: inherit the _compute_price method in product_pricelist model

In version 15.0 the _compute_price_rule method was used to obtain the
product price because the price was a computed field, however, in
version 16.0 we need to inherit _compute_price from product_pricelist_item
because this method is used directly when creating a new sale order line.

Commit:
[UPD] Update product_pricelist_supplierinfo.pot

OCA-git-bot committed:
product_pricelist_supplierinfo 16.0.1.0.1

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_pricelist_supplierinfo/

simahawk committed:
product_sale_manufactured_for: move rel to variant

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_pricelist_supplierinfo/es/

simahawk committed:
product_sale_manufactured_for: fix archived partner display

Before this change, customers that were archived couldn't be deleted from manufactured for partners.

OCA-git-bot committed:
[UPD] README.rst

rousseldenis committed:
[IMP] product_packaging_level: Improve README

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
oca-github-bot post-merge updates

rousseldenis committed:
[FIX] product_packaging_level: Use correct field in hook

OCA-git-bot committed:
oca-github-bot post-merge updates

Commit:
[UPD] Update product_supplierinfo_archive.pot

OCA-git-bot committed:
oca-github-bot post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (25 of 25 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

anothingguy committed:
[IMP] product_packaging_level: configure product packaging name

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (32 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

josep-tecnativa committed:
[IMP] product_category_active: test performance improvement
- Include context keys for avoiding mail operations overhead.

[BOT] post-merge updates

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (14 of 14 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_pricelist_supplierinfo/pt_BR/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_pricelist_supplierinfo/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_supplierinfo_archive
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_supplierinfo_archive/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 71.8% (23 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (32 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_category_active/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_attribute_archive
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_attribute_archive/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_sale_manufactured_for
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_sale_manufactured_for/it/

psc committed:
Translated using Weblate (German)

Currently translated at 84.6% (44 of 52 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_medical/de/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_attribute_archive
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_attribute_archive/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (52 of 52 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_medical/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_sale_manufactured_for
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_sale_manufactured_for/es/

Commit:
[16.0][FIX] product_packaging_level: Preserve sequence order

Commit:
[IMP]product_packaging_level: recover name_get policy flow

CarlosVForgeFlow committed:
[IMP] product_supplierinfo_archive: pre-commit auto fixes

CarlosVForgeFlow committed:
[MIG] product_supplierinfo_archive: Migration to 17.0

Commit:
[UPD] Update product_supplierinfo_archive.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (32 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (34 of 34 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (34 of 34 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (34 of 34 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

adriresu committed:
[IMP] product_main_supplierinfo: pre-commit auto fixes

adriresu committed:
[MIG] product_main_supplierinfo: Migration to 17.0

bizzappdev committed:
[IMP] product_packaging_level: pre-commit auto fixes

bizzappdev committed:
[MIG] product_packaging_level: Migration to 17.0

mdurepos committed:
[IMP] product_pricelist_supplierinfo: pre-commit auto fixes

mdurepos committed:
[MIG] product_pricelist_supplierinfo: Migration to 17.0

Commit:
[UPD] Update product_pricelist_supplierinfo.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-17.0/product-attribute-17.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_pricelist_supplierinfo/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_pricelist_supplierinfo/it/

uvid-alen committed:
Added translation using Weblate (Croatian)

Commit:
[IMP] product_medical: pre-commit auto fixes

Commit:
[MIG] product_medical: Migration to 18.0

chrisb-c01 committed:
[MIG] product_sequence: Migration to 18.0

natuan9 committed:
[MIG] product_state: Migration to 18.0

BhaveshHeliconia committed:
[MIG] product_pricelist_fixed_currency_rate: Migration to 18.0

Commit:
[MIG] product_sale_manufactured_for: Migration to 18.0

Commit:
[IMP] product_sale_manufactured_for: pre-commit execution

xaviedoanhduy commented on product_sale_manufactured_for/README.rst:
```diff
@@ -63,11 +63,13 @@ Contributors
 Other credits
 -------------
 
-The development of this module was financially supported by Camptocamp
```
please dont remove this line, you can add the new line above it

xaviedoanhduy reviewed (approved):
minor remark. could you try to drop commit `[ADD] add odoo-checklog` to see if the module without `tests` can cause CI to turn red again?

pre-approve

LucasTran380381 commented:
> minor remark. could you try to drop commit `[ADD] add odoo-checklog` to see if the module without `tests` can cause CI to turn red again?
> 
> pre-approve

I don't know why this repo not trigger CI

xaviedoanhduy commented:
dont worrry, I have checked it in my fork: https://github.com/xaviedoanhduy/product-attribute/pull/1

xaviedoanhduy reviewed (approved):
Looks good, thanks

trisdoan committed:
[IMP] product_packaging_level: pre-commit auto fixes

trisdoan committed:
[MIG] product_packaging_level: Migration to 18.0

trisdoan committed:
[FIX] product_packaging_level: cannot input name with user_defined policy

xaviedoanhduy committed:
[IMP] product_main_supplierinfo: pre-commit auto fixes

xaviedoanhduy committed:
[MIG] product_main_supplierinfo: Migration to 18.0

Commit:
product_main_supplierinfo: replace '_get_sellers' by std '_get_filtered_sellers'

Co-authored-by: Do Anh Duy <duyda@trobz.com>

natuan9 committed:
[IMP] product_state: Remove dependencies on stock and sale modules

OCA-git-bot committed:
Merge PR #1810 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_main_supplierinfo.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_main_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_main_supplierinfo/it/

OCA-git-bot committed:
Merge PR #1781 into 18.0

Signed-off-by StefanRijnhart

Commit:
[UPD] Update product_manufacturer.pot

OCA-git-bot committed:
[BOT] post-merge updates

victoralmau committed:
[IMP] product_category_active: pre-commit auto fixes

victoralmau committed:
[MIG] product_category_active: Migration to 17.0

TT52280

Commit:
[UPD] Update product_category_active.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1751 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update product_logistics_uom.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/

OCA-git-bot committed:
Merge PR #1793 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update product_pricelist_fixed_currency_rate.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_pricelist_fixed_currency_rate
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_pricelist_fixed_currency_rate/

OCA-git-bot committed:
Merge PR #1753 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_medical.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1758 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_set.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_medical/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_set/

pfranck committed:
[IMP] product_supplierinfo_archive: pre-commit auto fixes

Kimkhoi3010 committed:
[MIG] product_supplierinfo_archive: Migration to 18.0
Co-authored-by: Do Anh Duy <duyda@trobz.com>

BhaveshHeliconia committed:
[MIG] product_category_active: Migration to 18.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (51 of 51 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_medical/it/

OCA-git-bot committed:
Merge PR #1787 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update product_logistics_uom.pot

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (31 of 31 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/it/

OCA-git-bot committed:
Merge PR #1752 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update product_packaging_dimension.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1780 into 18.0

Signed-off-by StefanRijnhart

Commit:
[UPD] Update product_sequence.pot

OCA-git-bot committed:
[BOT] post-merge updates

legalsylvain committed:
[FIX] product_category_active: harmonization of category display.
In odoo, archived items are not displayed by default. (see users view, products, etc...)
The product_category_active module doesn't follow this convention because it is changing the context of the action of product.category. As a result, all categories are displayed, polluting visibility with obsolete items.
- Remove the active_test in the context of the action.
- create a migration script to remove the active_test on existing instances.
- add filters to display archived and 'archived or active' categories, in case you want to see all categories

CLaurelB committed:
[IMP] product_pricelist_supplierinfo: improve sale margin string

Update the sale margin string on the field to match the format of the
discount field [1], which is located beside it in the list view.

[1]: https://github.com/odoo/odoo/blob/e947c589/addons/product/models/product_supplierinfo.py#L65

CLaurelB committed:
[IMP] product_pricelist_supplierinfo: exclude supplier info discount in price

Introduce a new field to optionally exclude the discount set in the
product supplier info when calculating the product's price.
Additionally, add a test to validate the new functionality.

Commit:
[UPD] Update product_pricelist_supplierinfo.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-17.0/product-attribute-17.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_pricelist_supplierinfo/

CLaurelB committed:
[IMP] product_pricelist_supplierinfo: pre-commit auto fixes

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (15 of 15 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_pricelist_supplierinfo
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_pricelist_supplierinfo/it/

yankinmax committed:
[IMP] product_attribute_archive: pre-commit auto fixes

yankinmax committed:
[MIG] product_attribute_archive: Migration to 18.0

OCA-git-bot committed:
Merge PR #1806 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_packaging_level/

CLaurelB committed:
[MIG] product_pricelist_supplierinfo: Migration to 18.0

Changelog:
- Use `is` and `is not` for type comparisons, or `isinstance()` for
  isinstance checks.
- Add a test for the scenario where the pricelist is not based on the
  supplier info, ensuring the price is calculated as expected natively.
- Add a test for the scenario where there are no sellers linked to the
  product.
- Replace the currency rate update method by the native
  `setup_other_currency` method, which requires inheriting from
  `BaseCommon`.

Co-authored-by: mle <marie.lejeune@acsone.eu>

OCA-git-bot committed:
Merge PR #1851 into 18.0

Signed-off-by legalsylvain

Commit:
[UPD] Update product_category_active.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_category_active/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_category_active/it/

OCA-git-bot committed:
Merge PR #1825 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_supplierinfo_archive.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1838 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update product_attribute_archive.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_attribute_archive
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_attribute_archive/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_attribute_archive
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_attribute_archive/it/

OCA-git-bot committed:
Merge PR #1788 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update product_supplierinfo_revision.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_supplierinfo_revision
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_supplierinfo_revision/

OCA-git-bot committed:
Merge PR #1837 into 18.0

Signed-off-by moylop260

Commit:
[UPD] Update product_pricelist_supplierinfo.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_pricelist_fixed_currency_rate
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_pricelist_fixed_currency_rate/it/

OCA-git-bot committed:
[IMP] update dotfiles

OCA-git-bot committed:
Merge PR #1795 into 18.0

Signed-off-by rousseldenis

Commit:
[UPD] Update product_state.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_state
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_state/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (31 of 31 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_state
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_state/it/
